laplace/marglik_training.py

```diff
@@ -31,20 +33,22 @@ def marglik_training(
     prior_prec_init=1.,
     sigma_noise_init=1.,
     temperature=1.,
+    sigma_noise_fixed=None,
```

Comment:
Maybe make this bool and use `sigma_noise_init` as the fixed setting for it. Otherwise, it's unclear how they overwrite each other.